src/mcp_25625.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "mcp_can.h"
+
+class MCP_CAN_25625 : public MCP_CAN {
+public:
+    MCP_CAN_25625(pin_t cs, pin_t standby, SPIClass &spi=SPI, int speed=10);
```

Comment:
What does `int speed=10` represent?  MHz?

Reply:
Yes. It's copying the default from the base class.